scipy/stats/_sensitivity_analysis.py

```diff
@@ -301,12 +308,12 @@ def sobol_indices(
             func(f_A: np.ndarray, f_B: np.ndarray, f_AB: np.ndarray)
             -> Tuple[np.ndarray, np.ndarray]
 
-        with ``f_A, f_B`` of shape (s, n) and ``f_AB`` of shape (d, s, n).
+        with ``f_A, f_B`` of shape ``(s, n)`` and ``f_AB`` of shape ``(d, s, n)``.
```

Comment:
The line is going to be longer than 79 chars. Either make a new line or don't do this change. In this case it's ok to not render this.